### PR TITLE
[add] Navigation toggle

### PR DESCRIPTION
**Pull request by Dizolivemint**

### CODE REVIEW

**Comment by Dizolivemint** [comment]
> This is something I was planning on implementing as a lesson or workshop. Would you be interested in presenting this work at the next Meetup?

Sure, I'll give it a shot!

**Comment by Dizolivemint** [comment]
Don't think the way I did this was the best since it relies on a css class to work. Would using v-if  be better?

File: src/components/root/Header.vue [hunk 1]
```diff
@@ -3,6 +3,7 @@
     <div class="container">
       <router-link to="/" class="navbar-brand">{{ title }}</router-link>
       <button
+        v-on:click="toggleVisible"
```
Comment (Stegosource) [inline src/components/root/Header.vue]: For complex things, an event hanlder is great. In this case, I think we can do something as simple as:

v-on:click="isVisible = !isVisible"

What do you think?

File: src/components/root/Header.vue [hunk 2]
```diff
@@ -33,6 +38,17 @@ export default {
...
+  },
+  data: function () {
+    return {
+      isVisible: false
+    }
+  },
+  methods: {
+    toggleVisible() { 
+      this.isVisible = !this.isVisible
+      return !this.isVisible
```
Comment (Stegosource) [inline src/components/root/Header.vue]: If we follow the comment above, we may not need this method at all, but I'm curious about why there is a `return` in this function?

Comment (Dizolivemint) [reply src/components/root/Header.vue]: :see_no_evil: Habit maybe.

**Comment by Stegosource** [comment]
Woop! Nice work @Dizolivemint

**Comment by Dizolivemint** [comment]
Thanks!

**Pull request merged**